host/src/main.rs

```diff
@@ -39,35 +41,70 @@ async fn main() -> Result<()> {
     info!("  op-derive: {}", Digest::from(OP_DERIVE_ID));
     info!("  op-compose: {}", Digest::from(OP_COMPOSE_ID));
 
+    // Prevent internal prover from calling Bonsai on its own
```

Comment:
Is that still needed? I was hoping, that this is also fixed with #91

Reply:
yeah I think it should be okay since #91 does not instantiate the default prover